app/views/foreman_ansible/job_templates/ansible_collections_-_install_from_galaxy.erb

```diff
@@ -0,0 +1,32 @@
+<%#
+name: Ansible Collection - Install from Galaxy
+job_category: Ansible Galaxy
+description_format: Install collections '%{ansible_collections_list}' from Galaxy
+snippet: false
+template_inputs:
+- name: ansible_collections_list
+  required: true
+  input_type: user
+  description: "List of collections in Ansible Galaxy to install, separated by commas, e.g:
+    \"mysql, nginx\"\r\n\r\nThe default collections_paths is configured in /etc/ansible/ansible.cfg,
+    you may override it by filling the 'collections_path' input.
+    Click on \"Advanced\" to see it."
+  advanced: false
+- name: collections_path
+  required: false
+  input_type: user
+  description: A particular directory where you want the downloaded collections to be placed.
+  advanced: true
+provider_type: Ansible
+kind: job_template
+model: JobTemplate
+%>
+
+---
+- hosts: all
+  tasks:
+    - command: ansible-galaxy collection install {{ item }} -p <%= input('collections_path').present? ? input('collections_path') : '/etc/ansible/collections' %>
+      register: out
+      with_items:
+        - <%= input('ansible_collections_list').split(',').join(' ') %>
```

Comment:
this doesn't do what you intend it to do.
It actually runs with list including only one item that is a string "<col1> <col2> <col3>" so the loop will only run once.
It works though, because ansible `galaxy collection install <col1> <col2> <col3>` works, but we should eighter run that directly or generate the list properly :)

Reply:
good catch, I guess this should instead be something like
```erb
      with_items:
<%= input('ansible_collections_list').split(',').map { |col| "        - #{col}"}.join("\n") %>
```

or any other implementation :-)

Reply:
I think we should be fine with just

``yml
- command: ansible-galaxy collection install <%= input('ansible_collections_list').split(',').join(' ') %> -p ...
```

without the loop :) unless we want to split the outputs to work with them separatelly, but I don't see reason for that

Reply:
thanks @ezr-ondrej !
I tried that before, and got this:
Invalid collection name 'mrlesmithjr.system  dsglaser.cis_security', name must be in the format <namespace>.<collection>. Please make sure namespace and collection name contains characters from [a-zA-Z0-9_] only.", "stderr_lines": ["[WARNING]:

which is kinda strange, because it did work before, but maybe doing the loop is better.

Reply:
didn't you wrap it in quotes, so it was treated as one argument?
Do what works for you better, here it doesn't matter much, but loop is usually slower :)

Reply:
you are right, I removed the quotes. thanks.